sys-devel/gcc/gcc7-7.3.0_2018_05_01.recipe

```diff
@@ -142,6 +141,54 @@ sourceDir=$(pwd)
 relativeInstallDir="develop/tools$secondaryArchSubDir"
 installDir="$prefix/$relativeInstallDir"
 objectsDir=$(pwd)/../${portVersionedName}-obj
+gccLibDir=lib/gcc/$effectiveTargetMachineTriple/$gccVersion
+
+defineDebugInfoPackage gcc${secondaryArchSuffix} \
+	"$gccLibDir"/cc1 \
+	"$gccLibDir"/cc1plus \
+	"$gccLibDir"/collect2 \
+	"$gccLibDir"/lto1 \
+	"$gccLibDir"/lto-wrapper \
+	"$installDir/bin"/cpp \
+	"$installDir/bin"/g++ \
+	"$installDir/bin"/gcc \
+	"$installDir/bin"/gcc-ar \
+	"$installDir/bin"/gcc-nm \
+	"$installDir/bin"/gcc-ranlib \
+	"$installDir/bin"/gcov \
+	"$installDir/bin"/gcov-tool
+#	Do we need theese?
+#	"$gccLibDir"/*.a \
+#	"$developLibDir"/*.a
+
+# Is it even possible to define debuginfo package for subpackages?
```

Comment:
>  Is it even possible to define debuginfo package for subpackages?

I did it with LibRaw in 8dfa1405321090cae0636a18a822853deb8a2849, but I only created one common debuginfo package.

In case you would like to try the same trick but with several debuginfo packages, this is what I would suggest:
```
defineDebugInfoPackage gcc${secondaryArchSuffix}_fortran \
	"${installDir/gcc$secondaryArchSuffix/gcc${secondaryArchSuffix}_fortran}"/bin/gfortran \
	"${gccLibDir/gcc$secondaryArchSuffix/gcc${secondaryArchSuffix}_fortran}"/f951 \
	"${gccLibDir/gcc$secondaryArchSuffix/gcc${secondaryArchSuffix}_fortran}"/libgfortran.a
```

If creating multiple debuginfo packages does not work then I would suggest that you create a common debuginfo package.

Reply:
Please be reminded that `getPackagePrefix fortran` will get you the prefix of the `_fortran` subpkg, so you could instead do

```sh
defineDebugInfoPackage gcc${secondaryArchSuffix}_fortran \
	"$(getPackagePrefix fortran)/$relativeInstallDir"/bin/gfortran \
	"$(getPackagePrefix fortran)/$gccLibDir"/f951 \
	"$(getPackagePrefix fortran)/$gccLibDir"/libgfortran.a
```

Which is way shorter :)